docs/cookbook.adoc

```diff
@@ -145,6 +145,66 @@ The full set of filters and possible argument values is available in
 https://api.adoptium.net/q/swagger-ui/[the API documentation]
 and can be used to specify exactly the information you require.
 
+[#example-five]
+== Example Five: Download JDKs via the API using Powershell.
+
+Below is a sample script written in Powershell for Windows This script downloads and verifies the JDKs via the API, and performs checksum validation and notifications via email for the results. This script is provided as an example to demonstrate how the API can be used to automate tasks such as downloading and verifying the JDK.
+
+[source, powershell]
+----
+# Define Folders For Downloads
+$LinuxShareFolder = "<path to where the Linux builds will be copied to>"
+$WindowsShareFolder = "<path to where the Windows builds will be copied to>"
+# Define Releases To Download
+$Releases = @('8','11','17','21')
+$Releases | ForEach-Object {
+    $Release = $_
+    $Platforms = @('linux','windows')
+    $Platforms | ForEach-Object {
+        $Platform = $_
+        $Types = @('jdk','jre')
+        $Types | ForEach-Object {
+            $Type = $_
+            $ReleaseInfo = Invoke-WebRequest -Uri "https://api.adoptium.net/v3/assets/latest/$Release/hotspot?architecture=x64&image_type=$Type&os=$Platform&vendor=eclipse" -UseBasicParsing | ConvertFrom-Json
+            $CurrentDate = Get-Date -UFormat "%Y-%m-%dT%H:%M:%SZ"
+            $TimeDifference = New-TimeSpan -Start $ReleaseInfo.binary.updated_at -End $CurrentDate
+            
+                if ("$Platform" -eq "linux") {
```

Comment:
the linux and windows paths look pretty similar, can probably be extracted to common code

Reply:
Done, rationalised to a single path.

Reply:
I think the if and else are now identical, but both paths still exist, can they be brought down to 1?

Reply:
Yup, should be better now. Thanks!